spotify/models/user.py

```diff
@@ -28,6 +30,63 @@
 REFRESH_TOKEN_URL = "https://accounts.spotify.com/api/token?grant_type=refresh_token&refresh_token={refresh_token}"
 
 
+class TokenInfo:
+    def __init__(
+            self,
+            access_token: Optional[str] = None,
+            expires_in: Optional[int] = None,
+            token_type: Optional[str] = None,
+            refresh_token: Optional[str] = None,
+            scope: Optional[str] = None,
+            **kwargs
+    ):
+        self.access_token = access_token
+        self.expires_in = expires_in
+        self.token_type = token_type
+        self.refresh_token = refresh_token
+        self.scope = scope
+        self.expires_at = time.time() + self.expires_in if self.expires_in else None
+
+    def is_token_valid(self) -> bool:
+        if self.access_token and self.expires_at and self.expires_at > time.time():
+            return True
+        return False
+
+    def serialize(self):
+        return {
+            'access_token': self.access_token,
+            'expires_in': self.expires_in,
+            'expires_at': self.expires_at,
+            'scope': self.scope,
+            'refresh_token': self.refresh_token,
+            'token_type': self.token_type
+        }
+
+    @classmethod
+    def from_file(cls, file_path: Path):
```

Comment:
there should be a set minimum requirement for data from a json file, access_token, refresh_token and expires_in perhaps